kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosButton.vue

```diff
@@ -118,11 +118,19 @@ export default {
     },
     buttonDisabled() {
       if (this.comment) {
-        const commentOwnerId = this.comment.identity && this.comment.identity.id;
-        return commentOwnerId === eXo.env.portal.userIdentityId;
+        const commentOwnerId = this.comment.identity && this.comment.identity.id; 
+        if (commentOwnerId === eXo.env.portal.userIdentityId) {
+          return 'same'; 
+        } else if (!this.comment.identity.profile.dataEntity.enabled){
+          return 'disabled';        
+        }
```

Comment:
Use a boolean computed property and use it for the check , instead of checking on a String value 'disabled'